dev-docs/prebid-server-bidders.md

```diff
@@ -0,0 +1,132 @@
+---
+layout: page
+title: Prebid Server Bidders
+description: Get Started with Prebid Server
+pid: 28
+top_nav_section: dev_docs
+nav_section: prebid-server
+---
+<style>
+.output-workspace {
+  margin-top: 3em;
+}
+.bidder-selection {
+  margin-bottom: 1em;
+}
+.bidder-selection input {
+  margin-right: 0.5em;
+}
+.bidder-fetch-error {
+  margin-left: 40px;
+}
+</style>
+<script type="text/javascript" src="{{site.baseurl}}/assets/js/prebid-server-api.js"></script>
+<div class="bs-docs-section" markdown="1">
+
+# Prebid Server Bidders
+
+Prebid Server adapters aren't javascript, so not all Prebid.js adapters are available in Prebid Server, and vice versa. See [the docs](https://github.com/prebid/prebid-server/blob/master/docs/developers/add-new-bidder.md) for how to build a Prebid Server adapter.
+
+All the Prebid Server bidders are listed below. Select one, and then choose an option for more details.
+
+<fieldset id="bidder-selection" class="bidder-selection">
+  <legend>Choose a Bidder from the list</legend>
+</fieldset>
+
+<strong>Then select the option which applies to you...</strong>
+
+<span>I want to </span>
+<select id="purpose-dropdown">
+  <option value="bidder-params">see the bidder params for</option>
+  <option value="see-bugs">see open bugs involving</option>
+  <option value="file-bug">log a bug about</option>
+</select>
+<span id="selected-bidder"></span>
+<div id="output-workspace" class="output-workspace"></div>
+</div>
+<script type="text/javascript" async>
+(function() {
+    function addBidder(parentNode, bidder, checked) {
+      var inputNode = document.createElement("input");
+      inputNode.type = "radio";
+      inputNode.id = bidder + "-choice";
+      inputNode.name = "bidder";
+      if (checked) {
+        inputNode.checked = true;
+      }
+      parentNode.appendChild(inputNode);
+      var labelNode = document.createElement("label");
+      labelNode.htmlFor = inputNode.id;
+      labelNode.innerHTML = bidder;
+      parentNode.appendChild(labelNode);
+      parentNode.appendChild(document.createElement("br"));
+    }
+    function newOption(text) {
+      var element = document.createElement("option");
+      element.value = text;
+      element.innerHTML = text;
+      return element;
+    }
+    function newButton(link, text) {
+      var button = document.createElement("button");
+      button.type = "button";
+      button.innerHTML = text;
+      button.addEventListener("click", function() {
+        window.location = link;
```

Comment:
it would be nice if this opened a new tab instead of redirecting the current page

Reply:
done. good idea ^^